PolarisServer/Client.cs

```diff
@@ -130,14 +130,24 @@ private void HandleConnectionLost()
             IsClosed = true;
         }
 
-        public void SendPacket(byte[] blob)
+        public void SendPacket(byte[] blob, string name = null)
         {
             var typeA = blob[4];
             var typeB = blob[5];
             var flags1 = blob[6];
             var flags2 = blob[7];
+            string sendName = "";
```

Comment:
Assigned value not used in any code path. Also recommend preferring `string.Empty`